src/main/java/bio/terra/common/CloudPlatformWrapper.java

```diff
@@ -33,6 +33,10 @@ public static CloudPlatformWrapper of(CloudPlatform cloudPlatform) {
         }
     }
 
+    public boolean is(CloudPlatform cloudPlatform) {
+        return false;
+    }
```

Comment:
nit: Thoughts on naming this method something more descriptive?

Reply:
This is following the paradigm of the factory method `CloudPlatformWrapper.of`. I agree the name is a little obtuse, but am struggling to come up with a better concise name.

Reply:
I think `is()` is fine for how this class will be used; you could make it `isPlatform()` but that seems redundant to me. Another option is to change it (back?) to `getPlatform()`? e.g. `if (wrapper.getPlatform() == CloudPlatform.AZURE) {` vs `if (wrapper.is(CloudPlatform.AZURE) {`. In this case `is()` vs `get()` doesn't seem to be buying us much.

Reply:
If this method is kept, it should be `abstract` in the base class.

Reply:
I like `is()` exactly because it forces the logic to be contained in the wrapper class and encourages any switching to be done within the wrapper class. The intended use of `is()` is when you want to single out a cloud platform and make a decision, not branch between all cloud platforms.